.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Garak linting
+
+on: [workflow_dispatch]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install pylint
+        run: |
+          python -m pip install --upgrade pip
+          pip install pylint
```

Comment:
Instead of a separate install should we add this as a `lint` dependency see:
https://github.com/leondz/garak/blob/8cecf8d78e565ea9c44f2267130161fac09a222c/pyproject.toml#L76-L78
https://github.com/leondz/garak/blob/8cecf8d78e565ea9c44f2267130161fac09a222c/requirements.txt#L33-L34